### PR TITLE
Enhance edit mode

### PR DESCRIPTION
**Pull request by patnorris**
Redesigns the Edit Mode on a Space

Adds several options to include new items in the space and aims to simplify the user experience to make editing more intuitive

No other functionality should be affected.

Potential tests (not comprehensive as options now allow for a wide spectrum of edits):
Add an item from the library
Upload a glb model (other models are not supported)
Upload an image
Upload a video (make sure video can be played on click in the space)
Add an A-Frame entity and try to customize
Make sure all changes are persisted when clicking the Save button

### CODE REVIEW

**Comment by patnorris** [comment]
To be reviewed after
https://github.com/Bebb-Protocol-and-Apps/PWS/pull/34
and https://github.com/Bebb-Protocol-and-Apps/PWS/pull/30

File: src/PersonalWebSpace_frontend/helpers/space_helpers.js [hunk 1]
```diff
@@ -80,7 +81,79 @@ export const getStringForSpaceFromModel = (modelUrl) => {
...
         <a-sky color="#ECECEC"></a-sky>
 
-        <a-entity gltf-model="#model-glb" position="0 0 -5"></a-entity>
+        <a-entity gltf-model="url(${modelUrl}).glb" position="0 3 -6"></a-entity>
+      </a-scene>
+    </body>
+  </html>`;
+};
+
+export const getStringForSpaceFromUserUploadedModel = () => {
```
[the remark this answers is not in the document]
Comment (patnorris) [reply src/PersonalWebSpace_frontend/helpers/space_helpers.js:90]: this is the plain space (the empty preview scene) and we'll add the uploaded model later on programmatically to get the complete preview

[the remark this answers is not in the document]
Comment (patnorris) [reply src/PersonalWebSpace_frontend/helpers/space_helpers.js:90]: e.g. in CreateSpace and Space, we select this scene (let aScene = iframe.contentWindow.document.querySelector('#aSceneForModelPreview');) and then add a modelEntity

File: src/PersonalWebSpace_frontend/components/MediaContentPreview.svelte [hunk 2]
```diff
@@ -0,0 +1,47 @@
+<script>
+  import { getStringForSpaceFromImageFile, getStringForSpaceFromVideoFile } from "../helpers/space_helpers";
+
+  export let contentUrl;
+  export let contentFiles;
+  contentFiles = contentFiles;
+
+// Note: HTML as string in Svelte needs the ending script tag to be escaped (see https://github.com/sveltejs/svelte/issues/5810)
+  let mediaContentPreviewString
+  if (contentFiles[0].name.endsWith('.jpg')) {
```
[the remark this answers is not in the document]
Comment (patnorris) [reply src/PersonalWebSpace_frontend/components/MediaContentPreview.svelte:10]: good point, I'll push simplified code

**Comment by tiusty** [comment]
tested on alexStaging

**Review by tiusty — APPROVED** [review]

**Pull request merged**